migration/v0.15.x-v0.16.x/migrate-crd.sh

```diff
@@ -0,0 +1,106 @@
+            '{ "meta.helm.sh/release-name": "'"${release_name}"'",' \
+              '"meta.helm.sh/release-namespace": "'"${release_namespace}"'"},' \
+            '"labels":' \
+              '{ "app.kubernetes.io/managed-by": "Helm",'\
+                '"helm.sh/chart": "'"${chart}"'"}}}'\'
+}
+
+RELEASE_NAME="cert-manager"
+RELEASE_NAMESPACE="cert-manager"
+CHART="cert-manager-v1.1.0"
+CRDS=(  certificaterequests.cert-manager.io
+        certificates.cert-manager.io
+        challenges.acme.cert-manager.io
+        clusterissuers.cert-manager.io
+        issuers.cert-manager.io
+        orders.acme.cert-manager.io )
+
+for CRD in "${CRDS[@]}"; do
+  "${here}/../../bin/ck8s" ops kubectl wc patch crd "${CRD}" --patch "$(get_patch "${RELEASE_NAME}" "${RELEASE_NAMESPACE}" "${CHART}")"
+  "${here}/../../bin/ck8s" ops kubectl sc patch crd "${CRD}" --patch "$(get_patch "${RELEASE_NAME}" "${RELEASE_NAMESPACE}" "${CHART}")"
+done
+
+RELEASE_NAME="velero"
+RELEASE_NAMESPACE="velero"
+CHART="velero-2.15.0"
+
+CRDS=(  backups.velero.io
+        backupstoragelocations.velero.io
+        deletebackuprequests.velero.io
+        downloadrequests.velero.io
+        podvolumebackups.velero.io
+        podvolumerestores.velero.io
+        resticrepositories.velero.io
+        restores.velero.io
+        schedules.velero.io
+        serverstatusrequests.velero.io
+        volumesnapshotlocations.velero.io )
+
+for CRD in "${CRDS[@]}"; do
+  "${here}/../../bin/ck8s" ops kubectl wc patch crd "${CRD}" --patch "$(get_patch "${RELEASE_NAME}" "${RELEASE_NAMESPACE}" "${CHART}")"
+  "${here}/../../bin/ck8s" ops kubectl sc patch crd "${CRD}" --patch "$(get_patch "${RELEASE_NAME}" "${RELEASE_NAMESPACE}" "${CHART}")"
+done
+
+RELEASE_NAME="kube-prometheus-stack"
+RELEASE_NAMESPACE="monitoring"
+CHART="kube-prometheus-stack-12.8.0"
+CRDS=(  alertmanagerconfigs.monitoring.coreos.com
+        alertmanagers.monitoring.coreos.com
+        podmonitors.monitoring.coreos.com
+        probes.monitoring.coreos.com
+        prometheuses.monitoring.coreos.com
+        prometheusrules.monitoring.coreos.com
+        servicemonitors.monitoring.coreos.com
+        thanosrulers.monitoring.coreos.com )
+
+for CRD in "${CRDS[@]}"; do
+  "${here}/../../bin/ck8s" ops kubectl wc patch crd "${CRD}" --patch "$(get_patch "${RELEASE_NAME}" "${RELEASE_NAMESPACE}" "${CHART}")"
+  "${here}/../../bin/ck8s" ops kubectl sc patch crd "${CRD}" --patch "$(get_patch "${RELEASE_NAME}" "${RELEASE_NAMESPACE}" "${CHART}")"
+done
+
+# WC only
+
+RELEASE_NAME="gatekeeper"
+RELEASE_NAMESPACE="gatekeeper-system"
+CHART="gatekeeper"
+CRDS=(  configs.config.gatekeeper.sh
+        constraintpodstatuses.status.gatekeeper.sh
+        constrainttemplatepodstatuses.status.gatekeeper.sh
+        constrainttemplates.templates.gatekeeper.sh
+        k8sallowedrepos.constraints.gatekeeper.sh
+        k8srequirenetworkpolicy.constraints.gatekeeper.sh
+        k8sresourcerequests.constraints.gatekeeper.sh )
```

Comment:
Maybe these shouldn't actually be here, since they are not actually created by helm, instead they are controlled directly by gatekeeper (they are created when you create the connecting constrainttemplates). Though it might be fine for them to be here.